telethon/tl/custom/button.py

```diff
@@ -73,6 +73,10 @@ def inline(text, data=None):
 
         return types.KeyboardButtonCallback(text, data)
 
+    @staticmethod
+    def login(text, url, bot, request_write_access=None, fwd_text=None):
```

Comment:
Missing documentation with all of our current findings.

Reply:
Do you think these parameter names are the best? If we can make them more friendly, we should.

Reply:
`bot` won't work if I pass a username, although ideally we would want it to work. At worst, we should call `utils.get_input_user` here so that it doesn't need to be done in user code.

Reply:
> Do you think these parameter names are the best? If we can make them more friendly, we should.

I used same names as in Telegram docs